### ISSUE TITLE
Question: Tabview - why selected index is not updated when a Tab is deleted on the left from the selected tab

### ISSUE THREAD

**Issue by beervoley** [microsoft/microsoft-ui-xaml#4450]
@chingucoding Hi Marcel, I wanted to ask why the selected index is not update when a tab was deleted on the left from the selected tab?
The selected tab doesn't change - but the index does. Shouldn't it be something like this?
![image](https://user-images.githubusercontent.com/21356912/110371854-6653f400-8002-11eb-945d-be631fb585d4.png)

**Comment by chingucoding** [comment]
Yup, it should do that but doesn't so we end up with a faulty selected index. This is a bug.
